applications/ConstitutiveLawsApplication/custom_processes/advance_in_time_high_cycle_fatigue_process.cpp

```diff
@@ -84,6 +86,87 @@ void AdvanceInTimeHighCycleFatigueProcess::Execute()
 /***********************************************************************************/
 /***********************************************************************************/
 
+void AdvanceInTimeHighCycleFatigueProcess::MonotonicOrCyclicLoad()
+{
+    auto& process_info = mrModelPart.GetProcessInfo();
+    double time = process_info[TIME];
+    double delta_time;
+    if (mThisParameters["solver_settings"]["time_stepping"].Has("time_step")) {

```

Comment:
This is wrong IMO. We should take the current time step from the process info as `r_process_info[DELTA_TIME]`